airflow/operators/docker_operator.py

```diff
@@ -72,6 +72,9 @@ class DockerOperator(BaseOperator):
     :param mem_limit: Maximum amount of memory the container can use.
         Either a float value, which represents the limit in bytes,
         or a string like ``128m`` or ``1g``.
+    :param host_tmp_dir: Specify the location of the temporary directory on the host which will
+        be mapped to tmp_dir. If not provided defaults to using the standard system temp directory.
+    :type host_tmp_dir: str
```

Comment:
This addtion "splits" the description of `mem_limit` from it's type - these lines should be added before the one starting `:param mem_limit`